firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/FirebaseAppDistribution.java

```diff
@@ -346,19 +346,11 @@ private UpdateTaskImpl showUpdateAlertDialog(AppDistributionRelease newRelease)
     updateDialog.setButton(
         AlertDialog.BUTTON_NEGATIVE,
         context.getString(R.string.update_no_button),
-        (dialogInterface, i) -> {
-          dialogInterface.dismiss();
```

Comment:
I notice all of these `dialogInterface.dismiss()` calls are gone. Were they unnecessary?

Reply:
Awesome, thanks!